src/Tizen.NUI/src/public/WebView/WebSettings.cs

```diff
@@ -201,7 +202,7 @@ public bool JavaScriptEnabled
         /// <summary>
         /// Enable auto fitting
```

Comment:
previous ones were "Enables ... or not", the next ones are "Enable ..." - please use "Enables" consistently

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520